src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -123,4 +123,21 @@ public void doGet(
     }
     return events;
   }
+
+  @Override
+  public void doPost(
+      HttpServletRequest request, HttpServletResponse response, Credential googleCredential)
+      throws IOException {
+    assert googleCredential != null
+        : "Null credentials (i.e. unauthenticated requests) should already be handled";
+
+    CalendarClient calendarClient = calendarClientFactory.getCalendarClient(googleCredential);
+    Date start = new Date(request.getParameter("start"));
+    Date end = new Date(request.getParameter("end"));
+    String summary = "Read emails";
+    String calendarId = "primary";
```

Comment:
What's the meaning of "primary" here? Is this a placeholder or does this have a special meaning to Google Calendar?

Reply:
It is the Id of the primary calendar. We are using the primary calendar because it is common for all users, where as other calendars might not be present for all users (eg Canadian Hollidays Calendar).

Reply:
This should be a class constant.